_docs/request-matching.md

```diff
@@ -987,6 +987,46 @@ The full list of comparison types used by default is as follows:
 `CHILD_LOOKUP`
 `ATTR_NAME_LOOKUP`
 
+#### Same child nodes with different content
+By default WireMock takes into account an order of identical child nodes. Meaning if actual request has different order of same node on same level than stub it won't be matched. 
+This can be changed by passing additional argument to the `equalToXml` method
```

Comment:
Your code PR will be going out in WireMock version `3.7.0`  Would it be possible to add a small update to this line to let people know which version of WireMock they can use the new argument ?
```suggestion
As of WireMock version `3.7.0`, this can be changed by passing additional argument to the `equalToXml` method
```

Reply:
Added your suggestion and also aligned the examples for Java and JSON